ngo_mob.html

```diff
@@ -0,0 +1,556 @@
+<html>
```

Comment:
why a separate html for the mobile?  can't the original html modified to support the mobile resolution as well ?

Reply:
Created a new file, since the old file(ngo.html) was not correct.
ngo_mob.html has both desktop n mobile view

Reply:
If the old file was not correct, it is better to fix it rather than creating a new one.  For an external person, now it looks like there are 2 files and he/she will not know why.

Reply:
shall i delete the old file?

Reply:
yes.  And rename the latest to just ngo.html

Reply:
renamed the new file to ngo.html